bootstrap/044_brokers.sql

```diff
@@ -0,0 +1,17 @@
+
+
+create or replace function admin.list_brokers()
+    returns TABLE (flow_name VARCHAR, flow_is_default VARCHAR, flow_default_warehouse VARCHAR, flow_hostname VARCHAR, flow_last_modified_at VARCHAR, flow_ddl VARCHAR, flow_results_path VARCHAR, flow_comment VARCHAR)
```

Comment:
The `flow_` prefix on all of these columns is a little odd (given we're in a `list_brokers` procedure). Low-pri: I think you could drop the prefix on all of them (`flow_name` -> `name`, `flow_is_default` -> `is_default`).

Reply:
I am good with dropping the prefix but pls change the column names in the DDL too [here](https://github.com/sundeck-io/sundeck/blob/main/services/query_service/sundeck/sundeck_flow_handler.go#L355) so that they are in-sync.